go.mod

```diff
@@ -3,32 +3,32 @@ module github.com/container-orchestrated-devices/container-device-interface
 go 1.17
 
 require (
+	github.com/fsnotify/fsnotify v1.5.1
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/opencontainers/runc v1.0.3
+	github.com/opencontainers/runc v1.1.2
 	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
 	github.com/opencontainers/runtime-tools v0.0.0-20190417131837-cd1349b7c47e
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.2.1
 	github.com/stretchr/testify v1.7.0
 	github.com/xeipuuv/gojsonschema v1.2.0
+	golang.org/x/sys v0.0.0-20211116061358-0a5406a5449c
 	gopkg.in/fsnotify.v1 v1.4.7
 	sigs.k8s.io/yaml v1.3.0
 )
 
 require (
 	github.com/blang/semver v3.5.1+incompatible // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/fsnotify/fsnotify v1.5.1 // indirect
 	github.com/hashicorp/errwrap v1.0.0 // indirect
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
-	github.com/opencontainers/selinux v1.9.1 // indirect
+	github.com/opencontainers/selinux v1.10.0 // indirect
```

Comment:
@klihub this was an issue in the past. Could you test this PR?

Reply:
`K8s` master HEAD has been updated to 1.10.0 and `cri-o` main HEAD to 1.10.1. So I guess bumping our deps to should be fine now, too.

Reply:
OK. That's good news. Feel free to merge this PR then.